includes/classes/Feature/Facets/Facets.php

```diff
@@ -255,7 +255,7 @@ public function facet_query( $query ) {
 		foreach ( $taxonomies as $slug => $taxonomy ) {
 			$facets[ $slug ] = array(
 				'terms' => array(
-					'size'  => 10000,
+					'size'  => apply_filters( 'ep_facet_taxonomies_size', 5 ),
```

Comment:
Can we pass the taxonomy name through to the filter?
```suggestion
					'size'  => apply_filters( 'ep_facet_taxonomies_size', 5, $taxonomy ),
```